motoko/icrc2-swap/test/deploy.sh

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env bash
+
+set -e
+
+canister_exists() {
+  (dfx canister info "$1" 2>&1 | grep 'Module hash: 0x')
+}
+
+echo
+echo == Create Minting Account.
+echo
+
+if (dfx identity list | grep icrc2-swap-minter 2>&1 >/dev/null) ; then
+    echo "icrc2-swap-minter account already exists" >&2
+else
+    dfx identity import icrc2-swap-minter <(cat <<EOF
```

Comment:
```suggestion
    dfx identity import icrc2-swap-minter -- storage-mode plaintext <(cat <<EOF
```
CI had a problem with passwords. This should fix it